RecoDataProducts/inc/TrkStrawHitSeed.hh

```diff
@@ -9,67 +9,142 @@
-    Float_t     wireRes() const { return _werr; }
-    Float_t     energyDep() const { return _edep; }
-    StrawEnd    driftEnd() const { return _end; }
+    Float_t wireDOCA() const { return _rdoca; }
     Int_t ambig() const { return _ambig; }
-    StrawHitFlag const& flag() const { return _flag; }
+    //
+    //  Payload
     //
     StrawHitIndex   _index;       // index to the original straw (Combo) hit, and (for MC) MCDigi
-    StrawId     _sid;   // which straw has the hit
-    HitT0     _t0;    // time origin for this hit = track t0 + particle propagation time to this straw
-    Float_t     _trklen;    // Length from the nominal track start to the POCA of this hit
-    Float_t     _hitlen;    // Length from the straw center to the POCA of this hit
-    Float_t     _rdrift;    // drift radius for this hit
-    Float_t     _dtime;   // drift time from TOT for this hit
-    Float_t     _stime;   // signal propagation time for this hit, to the nearest end
-    Float_t     _htime;   // measured time for this hit
-    Float_t     _wdoca;   // DOCA from the track to the wire, signed by the angular momentum WRT the wire
-    Float_t     _rerr;    // intrinsic radial error
-    Int_t     _ambig;   // LR ambiguity assigned to this hit
-    Float_t         _edep;        // reco energy deposition
-    Float_t         _wdist;       // wire distance
-    Float_t         _werr;        // wire distance error estimate
-    StrawEnd      _end;         // straw end used for hit time measurement
+    StrawId         _sid;   // which straw has the hit
+    StrawEnd        _end;         // straw end used for hit time measurement
     StrawHitFlag    _flag;    // flag describing the status of this hit (active, ....)
+    Int_t           _ambig;   // hit state, including LR ambiguity
+    Int_t           _algo;     // hit updater algorithm
+    Bool_t          _frozen; // hit state was frozen
+    Float_t         _edep;        // reco energy deposition
+    Float_t         _htime;   // raw hit time
+    Float_t         _wdist;       // raw hit U position
+    Float_t         _werr;    // raw hit U position error estimate
+    Float_t         _tottdrift;   // drift time from TOT for this hit
+    float_t         _ptoca;    // reference particle time of closest approach (TOCA)
+    float_t         _stoca;    // reference sensor time of closest approach (TOCA)
+    Float_t         _rdoca, _rdocavar;   // reference (biased) DOCA from the track to the wire, signed by the angular momentum WRT the wire and the measurement end (and variance)
+    Float_t         _rdt, _rtocavar;   // reference (biased) time difference (and variance) at POCA
+    Float_t         _udoca, _udocavar; // unbiaed DOCA (and variance)
+    Float_t         _udt, _utocavar;   //unbiased dt and variance
+    Float_t         _rupos; // reference POCA position along the straw WRT the straw middle
+    Float_t         _uupos; // unbiased POCA position along the straw WRT the straw middle
```

Comment:
_uupos is computed but it has no accessor.  Is one needed?

Reply:
The accessor functions are legacy.  Updated and future development uses the struct data members directly, which I find clearer and less maintenance.